doc/module_examples/04_fix_issue/check.sh

```diff
@@ -0,0 +1,168 @@
+#!/bin/bash
+
+. /usr/share/preupgrade/common.sh
+
+#END GENERATED SECTION
+
+#
+# The whole section above is processed and modified by the preupg-xccdf-compose
+# script, according to the contents of the INI file (in this case it is the
+# content.ini file). In addition, the LICENSE used by the Preupgrade Assistant
+# is inserted.
```

Comment:
This note is duplicate. Because this is a tutorial and the user goes through all the parts, starting with 01, I think it can be specified just in the 01 part.